Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version:5.2
+
+import PackageDescription
+
+let package = Package(
+    name: "UIScreenExtension",
+    platforms: [
+        .iOS(SupportedPlatform.IOSVersion.v9),
+        .tvOS(SupportedPlatform.TVOSVersion.v9),
+        .watchOS(SupportedPlatform.WatchOSVersion.v2)
```

Comment:
I'm not sure about these two? :)

Reply:
Well, me neither to be honest 😳 The readme says that the extension supports tvOS and watchOS, so I added them as supported platforms to reflect the statement.

Practically, if clients have tvOS or watchOS apps they just can't use the package without these two lines.

Reply:
oh, hold on, my bad, the doc says it's for improvements. And they are're not supported indeed. Will remove the lines